### PR TITLE
Update create workspace topic; added delete workspace

### PR DESCRIPTION
**Pull request by pamgoodrich**

### CODE REVIEW

**Comment by pamgoodrich** [comment]
@marbemac : I added this to the bottom of the Create Workspace topic. I used the word "delete" in the section heading and text to help people who may search for this, but I used "remove" for the actual options.

File: 2.-workspaces/a.creating-a-workspace.md [hunk 1]
```diff
@@ -6,16 +6,37 @@ tags: [Workspaces]
...
 
-To create a new workspace, navigate to the [Stoplight homepage](https://stoplight.io/welcome) and select **Create a Stoplight workspace**.
+To create a new workspace:
 
-![create a workspace](../assets/images/create-worksapce-720.gif)
+1. Navigate to the [Stoplight home page](https://stoplight.io/)
 
-Once you have registered your workspace, you will be prompted to confirm your email address using a 6-digit verification code.
+2. Select **Get Started for Free**.
+
```
Comment (marbemac) [inline 2.-workspaces/a.creating-a-workspace.md:20]: Looks to be rendering OK - but just curious, any particular reason for new lines in between list items?

**Review by marbemac — APPROVED** [review]
looks good! just left a non-blocking question

File: 2.-workspaces/a.creating-a-workspace.md (hunk 1 above)
Comment (pamgoodrich) [reply 2.-workspaces/a.creating-a-workspace.md:20]: Ah... I missed this comment earlier. It's probably my natural instinct to type it that way. I'll watch for it going forward.

**Pull request merged**